integration/qbft/scenarios/integration_test_type.go

```diff
@@ -280,12 +257,6 @@ func (it *IntegrationTest) Run() error {
 					return fmt.Errorf("stored instance is nil, operator ID %v, instance index %v", operatorID, i)
 				}
 
-				jsonInstance, err := json.Marshal(storedInstance)
```

Comment:
Could you please keep it? It was also recently added to `integration-tests`. It's a debug log but I'm not sure if integration tests are stable enough in the CI, so if CI fails, this log may be useful. By the way, we probably need to add a to-do to remove it later after we make sure that integration tests are stable in the CI

Reply:
what do you mean? could you `suggest change` please?